### ISSUE TITLE
timeout does not work

### ISSUE THREAD

**Issue by cbergmann** [JoelGodOfwar/SinglePlayerSleep#7]
Hi there,

when one player goes to bed the message is shown but the time transition happens immediately and not after 10 seconds. This way the other playes have no chance of clicking cancel. Currently we are only two players to test and both are Operator/Admin with all permissions.

I use the current plugin Version and CraftBukkit version git-Bukkit-2012f1e (MC: 1.14.4) (Implementing API version 1.14.4-R0.1-SNAPSHOT)

Could you explain this behavior?

Kind Regards
Clemens

**Comment by JoelGodOfwar** [comment]
That is strange behavior, the code is designed to wait longer than the time needed by vanilla minecraft. Even the /sleep command uses a runnable to schedule the event after 10 seconds, that's so that if you are not on MP, or are the only one online, it will work as vanilla. Would you mind setting debug=true in the config.yml, and then try the plugin, and send me the log file containing the debug code?

**Comment by cbergmann** [comment]
Hi,

I have a log file here. Is it possible to send it to you in private?

**Comment by JoelGodOfwar** [comment]
send me a private message through  bukkit.org, https://dev.bukkit.org/projects/singleplayersleep2
That way your log will be between just the two of us.

**Comment by cbergmann** [comment]
Hi, Thanks for looking into this. I send you a Link to the Logfile via private Message.

**Comment by cbergmann** [comment]
Hi,

together with @JoelGodOfwar we could find the Problem. It was not a Problem with this Plugin but with the combination of two other plugins.

I disabled plugins one by one. I found out that the combination of EssentialsX and bpermissions lead to the problem. Then I looked through the [permissions page of essentialsx](https://essinfo.xeya.me/index.php?page=permissions) and found the essentials.sleepingignored permission. When I remove this Permissions from the group the sleep works as aspected.

Sorry for the hassle that was not related to your plugin.

Kind regards
Clemens
